src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 0.3s;
```

Comment:
Let's apply transition only to the required properties rather than to everything.

Reply:
got it